gateway/ln-gateway/src/events.rs

```diff
@@ -0,0 +1,51 @@
+use fedimint_core::core::ModuleKind;
+use fedimint_core::Amount;
+use fedimint_eventlog::{Event, EventKind};
+use fedimint_lnv2_common::contracts::{Commitment, OutgoingContract};
+use serde::{Deserialize, Serialize};
+
+use crate::sha256::Hash;
+
+#[derive(Serialize, Deserialize)]
+pub struct OutgoingPaymentSuccess {
+    pub outgoing_contract: OutgoingContract,
+    pub min_contract_amount: Amount,
+    pub invoice_amount: Amount,
```

Comment:
Discussed this in the call today, since it is redundant data we won't include it. We already have the payment hash and can lookup LN transaction information using that.